tools/oesgx/oesgx.c

```diff
@@ -31,6 +31,17 @@ void dump_regs(Regs* regs)
     printf("edx = 0x%x\n", regs->edx);
 }
 
+static void oe_get_cpuid(
```

Comment:
There are a few alternatives here:

1. We could leave a comment at the top of this function letting people know that it's a direct copy from cpuid.c.
2. We could call __cpuid_count directly, instead of calling oe_get_cpuid (my favorite solution)
3. We could forego defining the oe_get_cpuid function in this file and simply link against the object file "cpuid.o" as opposed to the entire oehost library (which has several library dependencies, which makes oesgx less useful/portable).

Reply:
Yeah, I'd just call __cupid_count directly.

Reply:
Please don't call `__cpuid_count` directly. One of the reasons `oe_get_cpuid()` exists is because it abstracts out the difference between Linux and Windows (which uses `__cpuidex`) 

There was an effort a while back to minimize the number of `ifdefs` strewn through code (we had like 3 or 4 different variants of `cpuid` based on different intrinsics or custom assembly) Is there a specific failure driving this or can we take the time to figure out a better way to share this (e.g. converting `oe_get_cpuid` into an inline header function)

Reply:
I'd be good with changing oe_get_cpuid into an inline header function. I think the Windows/Linux point is a good one and something important to consider when deciding on this

Reply:
@johnkord agree, that is the right thing to do. I will update the PR